robolectric/src/main/java/org/robolectric/RobolectricTestRunner.java

```diff
@@ -330,7 +339,7 @@ protected AndroidManifest getAppManifest(Config config) {
       AndroidManifest appManifest;
       appManifest = appManifestsByFile.get(identifier);
```

Comment:
The pull has been updated to include the package name as part of the cache identifier.